lightblue-client-integration-test/src/test/java/com/redhat/lightblue/client/integration/test/LockingTest.java

```diff
@@ -32,13 +36,43 @@ public void testAcquireAndRelease() throws Exception {
         assertTrue(lbLocker.release("test-lock"));
     }
 
+    @Test(expected = LightblueResponseException.class)
+    public void testPingWithError() throws Exception {
+        Locking lbLocker = getLightblueClient().getLocking(LOCK_DOMAIN);
+
+        lbLocker.ping("test-lock");
+    }
+
+    @Test(expected = LightblueResponseException.class)
+    public void testCountWithError() throws Exception {
+        Locking lbLocker = getLightblueClient().getLocking(LOCK_DOMAIN);
+
+        lbLocker.getLockCount("test-lock");
+    }
+
+    @Test(expected = LightblueResponseException.class)
+    public void testReleaseWithError() throws Exception {
+        Locking lbLocker = getLightblueClient().getLocking(LOCK_DOMAIN);
+
+        lbLocker.release("test-lock");
+    }
+
     @Test
     public void testLock() throws LightblueException, IOException {
         Locking lbLocker = getLightblueClient().getLocking(LOCK_DOMAIN);
         try (Lock lock = lbLocker.lock("test-lock")) {
             assertEquals(1, lock.getLockCount());
         }
-        assertEquals(0, lbLocker.getLockCount("test-lock"));
+
+        //assert the lock was successfully released.
+        try {
+            lbLocker.getLockCount("test-lock");
+            fail();
+        } catch (LightblueResponseException e) {
+            Error error = e.getLightblueResponse().getLightblueErrors()[0];
+            assertNotNull(error);
+            assertTrue(error.getMsg().contains("InvalidLockException: test-lock"));
+        }
```

Comment:
I mostly just want to highlight this consequence, but here is a case where not swallowing the exception makes for uglier client side code.